COVID_HUNGARY_REPORT.wls

```diff
@@ -1028,234 +640,164 @@ PlotStyle->{
 {RGBColor[0.8823529411764706`,0.2549019607843137`,0.07058823529411765`], Thickness[0.0075]},
 {RGBColor[0.1764705882352941`,0.2823529411764706`,0.39215686274509803`], Thickness[0.0075]},
 {RGBColor[0.06666666666666667`,0.06666666666666667`,0.09803921568627451`], Thickness[0.0075]}},
-PlotLegends->ageStructureLegend, AxesStyle->Directive[Gray,FontSize->20], LabelStyle->Directive[Bold, 20],
-Filling->{
+AxesStyle->Directive[Thickness[0.005],Black,FontSize->35], LabelStyle->Directive[Black,35],
+Frame->{{True, False},{True, False}},FrameStyle-> Directive[Thickness[0.005], 35],FrameLabel->{"days passed",""},Filling->{
 {7-> {{6}, RGBColor[0.06666666666666667`,0.06666666666666667`,0.09803921568627451`]}},
 {6-> {{5}, RGBColor[0.1764705882352941`,0.2823529411764706`,0.39215686274509803`]}},
 {5-> {{4}, RGBColor[0.8823529411764706`,0.2549019607843137`,0.07058823529411765`]}},
 {4-> {{3}, RGBColor[0.9372549019607843`,0.4627450980392157`,0.19215686274509802`]}},
 {3-> {{2}, RGBColor[0.9568627450980391`,0.7607843137254902`,0.3254901960784314`]}},
 {2-> {{1}, RGBColor[0.2549019607843137`,0.7176470588235294`,0.7058823529411764`]}},
 {1-> {Axis, RGBColor[0.615686274509804`,0.8196078431372549`,0.7333333333333333`]}}},
-Ticks->{{{0,"1.May"},{31+30+31+31,"1.September"},{31+30+31+30+31+30+31+31,"1.February"}},
-{{1000000,"1 Million"},{2000000,"2 Million"},{3000000,"3 Million"},{4000000,"4 Million"},{5000000,"5 Million"},{6000000,"6 Million"},{7000000,"7 Million"}}}]
+FrameTicks->{{0,120,240,360,480,600},{{300000,"300,000 "},{600000,"600,000 "},{900000,"900,000 "},{1200000,"1.2 Mio. "},{2500000,"2.5 Million"}}},Background->White,ImageSize->900]
 
 
-Export[StringJoin[generated,"school_cumu.pdf"],plt]
+(* ::Subsubsection:: *)
+(*Calculation for Table 2*)
 
 
-(* ::Subsection::Closed:: *)
-(*Elderly protection: R = 1.32*)
+FindMaximum[cumulative[[7]]/.sol,{t,70}]//Quiet
+
+
+Export[StringJoin[generated,"strong_cumu.pdf"],plt]
 
 
-(* ::Subsubsection::Closed:: *)
+(* ::Section::Closed:: *)
+(*Age-specific analysis*)
```

Comment:
SImilarly to previous section, please arrange sections here too:
Age-specific analysis
├── Plots for Figure 8.
 |       ├── Contact matrix
 |       ├── Hospital beds
 |       ├── ICU needs
 |       └── Mortality
└── Plots for Figure 9.
         ├── Contact matrix
         ├── Hospital beds
         ├── ICU needs
         └── Mortality